pkg/dockerregistry/server/quotarestrictedblobstore.go

```diff
@@ -126,6 +127,47 @@ func admitBlobWrite(ctx context.Context, repo *repository, size int64) error {
 	if size < 1 {
 		return nil
 	}
+
+	err := admitQuotas(ctx, repo, size)
+	if err != nil {
+		return err
+	}
+
+	err = admitLimitRanges(ctx, repo, size)
```

Comment:
updated in feedback commit of https://github.com/openshift/origin/pull/7989/commits/4a54ec640606eba808c536d233398ab63532db33